ios/RNTrackPlayer/Logic/MediaWrapper.swift

```diff
@@ -19,6 +19,7 @@ protocol MediaWrapperDelegate: class {
 class MediaWrapper: AudioPlayerDelegate {
     private var queue: [Track]
     private var currentIndex: Int
+    private var _rate: Float
```

Comment:
Is there a need for this? can't we just `set` or `get` `player.rate`?

Reply:
It does seem a bit silly. I'll remove 👍